plugins/modules/credential.py

```diff
@@ -175,11 +192,14 @@ def main() -> None:
 
     name = module.params.get("name")
     new_name = module.params.get("new_name")
+    copy_from = module.params.get("copy_from")
```

Comment:
Is this duplicate intended? What's the difference of this to the value of `copy_from` we already got in the line `161`?

Reply:
good catch. Updated.